sensor/kubernetes/fake/fake.go

```diff
@@ -85,14 +85,33 @@ type WorkloadManager struct {
 	networkManager      manager.Manager
 }
 
+// WorkloadManagerConfig WorkloadManager's configuration
+type WorkloadManagerConfig struct {
+	workloadFile string
+	client       client.Interface
+}
+
+// ConfigDefaults default configuration
+func ConfigDefaults() *WorkloadManagerConfig {
```

Comment:
How about renaming this to `NoFakeWorklod()` to highlight that the default behavior is not to use any fake workload?

Reply:
I see where you're coming from but I think that renaming this function to `NoFakeWorkload` could lead to a misunderstanding. `ConfigDefaults` takes the default path as `/var/scale/stackrox/workload.yaml` from `workloadPath`. The decision whether we use fake workloads or not is based on the existence of the configured file (whether it is the default file previously mentioned or a configured one using `WithWorkloadFile`)

Reply:
Ah, I see. Okay then 👍